File: plot/src/main/java/net/fwitz/math/main/escape/halley/HalleysMethodZ8p15Z4m16.java
package net.fwitz.math.main.escape.halley;

import net.fwitz.math.main.complex.analysis.examples.simple.Z8p15Z4m16;
import net.fwitz.math.main.escape.newton.NewtonsMethodZ8p15Z4m16;

public class HalleysMethodZ8p15Z4m16 extends NewtonsMethodZ8p15Z4m16 {
    public static void main(String[] args) {
        HalleysMethodPlot.render(
                "z^8 + 15z^4 - 16  (Halley's Method)",
                Z8p15Z4m16::f,
                Z8p15Z4m16::df,
                Z8p15Z4m16::d2f);
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/escape/halley/HalleysMethodZ8p15Z4m16.kt
package net.fwitz.math.main.binary.escape.halley

import net.fwitz.math.main.binary.complex.functions.examples.simple.Z8p15Z4m16
import net.fwitz.math.main.binary.escape.newton.NewtonsMethodZ8p15Z4m16

object HalleysMethodZ8p15Z4m16 : NewtonsMethodZ8p15Z4m16() {
    @JvmStatic
    fun main(args: Array<String>) = HalleysMethodPlot.render(
        name = "z^8 + 15z^4 - 16  (Halley's Method)",
        f = Z8p15Z4m16::f,
        df = Z8p15Z4m16::df,
        d2f = Z8p15Z4m16::d2f
    )
}